h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -254,6 +253,22 @@ public int compare(Integer o1, Integer o2) {
     }
   } // GLMModelOutputV2
 
+  public static void calculateVarimpBase(double[] magnitudes, Integer[] indices, double[] betaNorm) {
+    for (int i = 0; i < magnitudes.length; ++i) {
+      magnitudes[i] = (float) betaNorm[i];
```

Comment:
is there a reason to convert to float?

Reply:
For some reason VarImp class only takes float[] for values and String[] for names.